gp_align/align.py

```diff
@@ -7,36 +26,40 @@
 
 
 def align_plates(plate_image, calibration_plate):
-    """Both images should be converted to edges
-    Returns a vector (x, y) that describes the translation from the calibration plate to the analyte plate"""
+    """
+    Both images should be converted to edges
+    Returns a vector (x, y) that describes the translation from the calibration
+    plate to the analyze plate.
+    """
     r = int(RADIUS)
     best_offset = (0, 0)
     best_value = 0
-    for i in np.linspace(-r, r, 2*r+1).astype(int):
-        for j in np.linspace(-r, r, 2*r+1).astype(int):
+    # TODO: Potentially use vectorized numpy functions here.
```

Comment:
I guess a convolution could be used, but I don't know how much it would speed up as most of the time is spent doing `overlap = (image1_slice & image2_slice).sum()`